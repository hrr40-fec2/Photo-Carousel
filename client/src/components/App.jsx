import React from 'react';
import Carousel from './Carousel.jsx';
import $ from 'jquery';


export default class App extends React.Component {
  constructor(props) {
    super(props)
    this.state = {
      images: []
    }
  }


  render() {
    return (
      <Carousel />
    )
  }
};