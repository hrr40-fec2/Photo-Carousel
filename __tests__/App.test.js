import { shallow, mount } from 'enzyme';
import React from 'react';
import App from '../client/src/components/App.jsx';
import Tests from '../client/src/setupTests.js';
import sinon from 'sinon';

describe("App", () => {
  test("it properly renders", () => {
    const app = shallow(<App />);
    expect(app.getElements()).toMatchSnapshot();
  });

  test("it calls componentDidMount", () => {
    var spy = sinon.spy(App.prototype, 'componentDidMount');
    const wrapper = shallow(<App />);
    wrapper.instance().componentDidMount();
    expect(spy).toHaveBeenCalled;
  });

})