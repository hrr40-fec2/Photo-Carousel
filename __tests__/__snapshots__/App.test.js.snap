// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App it properly renders 1`] = `
Array [
  <ForwardRef(styled.div)>
    <Carousel
      photos={Array []}
    />
  </ForwardRef(styled.div)>,
]
`;
